src/ontology/ro-edit.owl

```diff
@@ -5329,6 +5331,27 @@ AnnotationAssertion(rdfs:label obo:RO_0002801 "co-roosts with"@en)
 SubObjectPropertyOf(obo:RO_0002801 obo:RO_0008506)
 SymmetricObjectProperty(obo:RO_0002801)
 
+# Object Property: obo:RO_0002802 (reservoir host of)
+
+AnnotationAssertion(obo:IAO_0000115 obo:RO_0002802 "A reservoir is a population, species or community (assemblage of different species in a given geographic area) in which a microorganism naturally occurs and is indefinitely maintained. Some zoonotic pathogens, particularly bacterial pathogens, may also have environmental reservoirs. Microorganisms with multiple reservoir species may be indefinitely maintained across the community of species even if they are not always present in each individual reservoir species. In the reservoir species or community, the microorganism may cause either asymptomatic infection or disease and this may vary among individuals. A pathogen may also be more genetically diverse in its reservoir host than in other taxa, in part because the pathogen is endemic to reservoir hosts.")
+AnnotationAssertion(obo:IAO_0000112 obo:RO_0002802 "An individual of species Camelus dromedarius (commonly known as dromedary camels) is a reservoir host of an individual of Middle East respiratory syndrome–related coronavirus (MERS-CoV) since at least the 1980s.")
+AnnotationAssertion(terms:contributor obo:RO_0002802 <https://orcid.org/0000-0003-3138-4118>)
+AnnotationAssertion(terms:contributor obo:RO_0002802 <https://orcid.org/0000-0002-3120-4802>)
+AnnotationAssertion(obo:IAO_0000119 obo:RO_0002802 <https://doi.org/10.3390/v13071356>)
+AnnotationAssertion(rdfs:label obo:RO_0002802 "reservoir host of")
+SubObjectPropertyOf(obo:RO_0002802 obo:RO_0002453)
+InverseObjectProperties(obo:RO_0002802 obo:RO_0002803)
```

Comment:
This logical axiom should be on the property that says it's inverse.

Reply:
@anitacaron Thanks again for taking the time to comment and for humoring my relative naivete . 

with your comment:
> This logical axiom should be on the property that says it's inverse.

do you mean that I should move the axiom to the obo:RO_0002803? 

I'd appreciate suggestions, I am still catching up with the ontology jargon.

Reply:
@anitacaron I've moved the logical axiom to be on the property that says it's inverse as you suggested. Please review https://github.com/oborel/obo-relations/pull/757/commits/a7ea89c83d4741f35d57c63d1e649a6779d2de0e and resolve our conversation or make suggestion for further improvements.

Thanks for reviewing and for being patient with me.

Reply:
Thanks @jhpoelen!